### PR TITLE
MAL Manga Support Through Second Jikan

### PR DESCRIPTION
**Pull request by PhantomOffKanagawa**
A few quick and basic changes I made for a possible way to add manga support through essentially copy-pasting the anime API with Jikan and changing relevant lines as the Jikan API for manga is a one-word difference. Did my best to follow exactly how the MALAPI was setup to make it as consistent as possible and has been working for me through my quick run through of using it on my own. 

The only change unrelated to adding manga support was the .gitignore change as it allowed me to ignore built js files when committing so obviously not a feature important piece. I am also not completely sure how some of the typeMappings work but as they seem to be working to my view as the sub-type changes to the relevant type and show correctly in the search.

I wanted to posit this as a possible new feature as it was something I wanted and realized this was the best way to add it as it didn't exist already. I'm new to GitHub collaboration so sorry for any etiquette missteps.

### CODE REVIEW

**Comment by mProjectsCode** [comment]
Thanks for the PR. I am currently quite busy, but I will try to review the PR before the weekend.

File: src/api/apis/MALAPIManga.ts [hunk 1]
```diff
@@ -0,0 +1,131 @@
+import { APIModel } from '../APIModel';
+import { MediaTypeModel } from '../../models/MediaTypeModel';
+// import { MovieModel } from '../../models/MovieModel';
```
Comment (mProjectsCode) [inline src/api/apis/MALAPIManga.ts:3]: Consider removing this comment.

File: src/api/apis/MALAPIManga.ts [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+		if (fetchData.status !== 200) {
+			throw Error(`MDB | Received status code ${fetchData.status} from an API.`);
+		}
+
+		const data = await fetchData.json();
+		console.debug(data);
+		const result = data.data;
+
+		const type = this.typeMappings.get(result.type?.toLowerCase());
+			const model = new MangaModel({
```
Comment (mProjectsCode) [inline src/api/apis/MALAPIManga.ts:99]: I think there is something wrong here with the indentation.

**Review by mProjectsCode — COMMENTED** [review]
Looks good, except for these two minor things.

**Comment by mProjectsCode** [comment]
It would also be good to update the readme to reflect the changes made in this PR.

**Comment by PhantomOffKanagawa** [comment]
Sorry for the slow turnaround. Removed comment, fixed formatting, updated README.md and synced with latest version. It should be all good to go.

**Comment by mProjectsCode** [comment]
LGTM

**Pull request merged**